src/db/mongodb.rs

```diff
@@ -100,6 +110,44 @@ impl MongoDb {
     }
 }
 
+pub struct MongoDbCollection<T>(mongodb::Collection<T>);
+
+impl<T: Serialize> MongoDbCollection<T> {
+    /// Inserts many records and ignores duplicate key errors.
+    pub async fn insert_many_ignore_duplicates(
+        &self,
+        docs: impl IntoIterator<Item = impl Borrow<T>>,
+        options: impl Into<Option<InsertManyOptions>>,
```

Comment:
I think we should bake the unordered in instead of making it a parameter, the function does not really make sense, otherwise.

Reply:
It can still make sense. The unordered just specifies that we won't stop as soon as we hit an error.

Reply:
But if the insert is ordered Mongo will stop after hitting the first error. In which case I think it's bad to ignore the error.

Reply:
I think it makes the most sense to mimic the way we call the normal function in every way.